From 9c1595dc423970cf86ef371dd15df9b27d9baf81 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 20 Mar 2017 16:13:07 +0000
Subject: [PATCH] functions: fix abuild -A

fix so abuild -A does not exit with error if gitbase fails

this was introduced with the set -e
---
 functions.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/functions.sh.in b/functions.sh.in
index 3e1840c..925dc43 100644
--- a/functions.sh.in
+++ b/functions.sh.in
@@ -71,7 +71,7 @@ readconfig() {
 	[ -f "$ABUILD_CONF" ] && . "$ABUILD_CONF"
 	[ -f "$ABUILD_USERCONF" ] && . "$ABUILD_USERCONF"
 	APORTSDIR=${_APORTSDIR-$APORTSDIR}
-	gitbase=$(git rev-parse --show-toplevel 2>/dev/null) # already is -P
+	gitbase=$(git rev-parse --show-toplevel 2>/dev/null) || true # already is -P
 	if [ -d "$APORTSDIR" ]; then
 		APORTSDIR=$(cd "$APORTSDIR"; pwd -P)
 	elif [ -z "$APORTSDIR" ] && [ -d "$HOME/aports" ]; then
-- 
2.11.1

